test/integration/build-output/test/index.test.js

```diff
@@ -94,8 +94,8 @@ describe('Build Output', () => {
       expect(parseFloat(indexSize) - 265).toBeLessThanOrEqual(0)
       expect(indexSize.endsWith('B')).toBe(true)
 
-      // should be no bigger than 60.8 kb
-      expect(parseFloat(indexFirstLoad) - 61).toBeLessThanOrEqual(0)
+      // should be no bigger than 63.7 kb
+      expect(parseFloat(indexFirstLoad) - 63.7).toBeLessThanOrEqual(0)
```

Comment:
hate to add 3KB of weight just cause of this... any possible way we can being this down?